### PR TITLE
Feature/add contributions from contributor endpoint

### LINKED ISSUE #60: Route to get list of donations for a given contributor
This route should return a list of contributions for a given contributorID.  

Ensure route works wit Pagination. You can look at other existing routes, for an example. Other ideas are welcome.

### PR DESCRIPTION
**Pull request by benjaminEwhite**
Closes #60

### CODE REVIEW

**Comment by benjaminEwhite** [comment]
@SchaeStewart i followed pattern i saw in other endpoints re: count vs. full count, but it was unclear to me what is going on with that, and assumed it may be to satisfy client needs, which I did not run down.

File: src/components/App.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import Header from './header'
 import Footer from './footer'
-import Candidate from './Candidate'
+import Candidate from './candidate'
```
Comment (benjaminEwhite) [inline src/components/App.js:4]: Not sure why this is turning up in the diff given that #61 was already merged....

Comment (SchaeStewart) [reply src/components/App.js:4]: yeah, that's weird. I'm not worried about it though

File: server/api_doc.md [hunk 2]
```diff
@@ -211,3 +211,65 @@ response:
...
+## GET `/api/contributors/:contributorId/contributions`
+
+Retrieve contributions for a contributor id.
+
+Query Params (optional):
+
+- `offset` default value `0`
+- `limit` default value `50`
+
+Example: `/api/search/contributors/2423d2db-2dd1-493e-8bf6-d85a44c2a58d?limit=2&offset=0`
```
Comment (SchaeStewart) [inline server/api_doc.md:224]: Example endpoint doesn't match the actual route. `'/contributors/:contributorId/contributions'`

**Comment by SchaeStewart** [comment]
> @SchaeStewart i followed pattern i saw in other endpoints re: count vs. full count, but it was unclear to me what is going on with that, and assumed it may be to satisfy client needs, which I did not run down.

I'm doing that because when you use `offset` and `limit` you don't know how many records there are total. Basically it's just to keep the app ready for pagination. There may be a better way to do this, and I'm open to ideas. This just seemed like a quick and easy enough solution for now

**Comment by benjaminEwhite** [comment]
@SchaeStewart yeah i figured it was for pagination. Is it the case that there's not really a need for a `full_count` on individual items, but that field came along for the ride since it's added per row from the SQL query?   Fine by me if so, just wanted to understand.

**Pull request merged**